crates/core/src/client/errors.rs

```diff
@@ -74,6 +74,12 @@ impl<T, E: std::error::Error> From<ConversionError<T>> for EthApiError<E> {
     }
 }
 
+impl<E: std::error::Error> From<FromByteSliceError> for EthApiError<E> {
```

Comment:
boyscout commit to fix an issue discussed during a conversation about no more map_err